manifest.toml

```diff
@@ -59,6 +59,8 @@ ram.runtime = "100M"
     [resources.system_user]
     # Hard-coded...
     home = "/home/yunohost.app/__APP__"
+    allow_ssh = true
+    allow_email = true
```

Comment:
and we need to adapt `app.ini` config file for mail
```
[mailer]
ENABLED = true
SMTP_ADDR = 127.0.0.1
SMTP_PORT = 25
FROM = "Gitea" <__APP__@__DOMAIN__>
USER = __APP__
PASSWD = __MAIL_PWD__
SKIP_VERIFY = true
```

Reply:
My bad. Fixed.